cves/kernel/CVE-2014-3183.yml

```diff
@@ -84,14 +102,10 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 51217e69697fba92a06e07e16f55c9a52d8e8945
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Automatically taken from NVD references list, manually confirmed.
```

Comment:
I would say to add what was added and deleted within this commit for more clarity in this section, but this is not a requirement just a suggestion